src/TabsInternal/V2.elm

```diff
@@ -136,6 +136,7 @@ viewTab_ config index tab =
                          Attributes.disabled (not isSelected && tab.disabled)
                        , Aria.selected isSelected
                        , Role.tab
+                       , Attributes.type_ "button"
```

Comment:
Looking at the name of the function `buttonOrLink`, I was reminded that sometimes the tab is an anchor element rather than a button.

[Elllie example showing that the tab can be an anchor](https://ellie-app.com/jftVLNmTL9ca1)
<img width="892" alt="Screen Shot of example ellie" src="https://user-images.githubusercontent.com/8811312/183222414-ac9f6ce5-40bb-411b-aeff-1dc371bae16b.png">


You can see this behavior in use in a couple of places in the app, but probably the easiest one to get to is Mock Data, in the assignment-type tabs.
<img width="475" alt="" src="https://user-images.githubusercontent.com/8811312/183222638-fadfe7eb-2b80-4ece-9c4f-c6063c162097.png">

Perhaps we should only attach the button type when the tag is a button?